.github/workflows/tests.yml

```
@@ -4,6 +4,21 @@ on:
   push:
 
 jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_RUBYGEMS__PKG__GITHUB__COM: gocardless-robot-readonly:${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v3
```

v4 is the current version
```suggestion
      - uses: actions/checkout@v4
```